src/Documents.jl

```diff
@@ -433,7 +437,7 @@ doctest_replace!(block) = true
 
 function buildnode(T::Type, block, doc, page)
     mod  = get(page.globals.meta, :CurrentModule, Main)
-    dict = Dict{Symbol, Any}(:source => page.source, :build => page.build)
+    dict = Dict{Symbol, Any}(:source => page.source, :build => page.build, :workdir => page.workdir)
```

Comment:
This is unnecessary, no?

```suggestion
    dict = Dict{Symbol, Any}(:source => page.source, :build => page.build)
```

Reply:
It appears to be, yes!